src/box/xlog.h

```diff
@@ -620,6 +639,13 @@ xlog_close_reuse_fd(struct xlog *l, int *fd);
 int
 xlog_materialize(struct xlog *l);
 
+/**
+ * Same as xlog_materialize but works with the filename instead of the xlog
+ * structure. The filename buffer is updated accordingly.
+ */
+int
+xlog_materialize_filename(char *filename);
```

Comment:
I don't see much point in exposing `xlog_materialize_filename` and `xlog_format_filename`. These functions are trivial and I see nothing wrong if you format file names explicitly in the sort data writer.

Reply:
Done.